spec/support/page_objects/providers/show.rb

```diff
@@ -14,6 +15,8 @@ class Show < PageObjects::Base
       element :unregistered_user_data, ".unregistered-users"
       element :edit_user_data, ".registered-users .user-card a"
 
+      section :delete_a_user, PageObjects::Sections::UserCard, ".user-card", match: :first
```

Comment:
```suggestion
      sections :user_cards, PageObjects::Sections::UserCard, ".user-card"
```

Reply:
this has a clash with the private method, so I've retained the `delete_a_user for` it to work